src/components/notification-picker.tsx

```diff
@@ -0,0 +1,226 @@
+    model.notification?.enableNotification ?? true
+  );
+  const [sendToInput, setSendToInput] = useState<string>(
+    model.notification?.sendTo?.join(', ') || ''
+  );
+  const [includeOutput, setIncludeOutput] = useState<boolean>(
+    model.notification?.includeOutput || false
+  );
+
+  const enableNotificationChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const updatedEnableNotification = e.target.checked;
+    setEnableNotification(updatedEnableNotification);
+    modelChange({
+      ...model,
+      notification: {
+        ...model.notification,
+        enableNotification: updatedEnableNotification
+      }
+    });
+  };
+
+  const selectChange = useCallback(
+    (e: SelectChangeEvent<string>) => {
+      const newEvent = e.target.value;
+      if (!selectedEvents.includes(newEvent)) {
+        const updatedEvents = [...selectedEvents, newEvent];
+        setSelectedEvents(updatedEvents);
+        modelChange({
+          ...model,
+          notification: {
+            ...model.notification,
+            selectedEvents: updatedEvents
+          }
+        });
+      }
+    },
+    [selectedEvents, model, modelChange]
+  );
+
+  const sendToChange = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setSendToInput(e.target.value);
+  }, []);
+
+  const blur = useCallback(() => {
+    const emailArray = sendToInput
+      .split(',')
+      .map(email => email.trim())
+      .filter(email => email);
+    const updatedNotification = { ...model.notification, sendTo: emailArray };
+    modelChange({ ...model, notification: updatedNotification });
+  }, [sendToInput, model, modelChange]);
+
+  const keyDown = useCallback(
+    (e: React.KeyboardEvent) => {
+      if (e.key === 'Enter') {
+        e.preventDefault();
+        blur();
+      }
+    },
+    [blur]
+  );
+
+  const includeOutputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const updatedValue = event.target.checked;
+    setIncludeOutput(updatedValue);
+    modelChange({
+      ...model,
+      notification: {
+        ...model.notification,
+        includeOutput: updatedValue
+      }
+    });
+  };
+
+  const deleteSelectedEvent = useCallback(
+    (eventToDelete: string) => () => {
+      const updatedEvents = selectedEvents.filter(
+        event => event !== eventToDelete
+      );
+      setSelectedEvents(updatedEvents);
+      modelChange({
+        ...model,
+        notifications: { ...model.notification, selectedEvents: updatedEvents }
+      });
+    },
+    [selectedEvents, model, modelChange]
+  );
+
+  if (!notificationEvents.length) {
+    return null;
+  }
+
+  return (
+    <Stack size={2}>
+      <InputLabel>{trans.__('Notification')}</InputLabel>
+      <FormControlLabel
+        control={
+          <Switch
+            checked={enableNotification}
+            onChange={enableNotificationChange}
+          />
+        }
+        label={trans.__('Enable Notification')}
```

Comment:
Don't use Title Case, but instead Sentence case: `'Enable notification'`. This matches the convention for other field labels.

Reply:
Do this for the other fields as well, like "Send To" and "Include Output".